src/app.js

```diff
@@ -214,8 +214,13 @@ function run () {
     },
     getValue: (cb) => {
       try {
-        var comp = $('#value').val().split(' ')
-        cb(null, executionContext.web3().toWei(comp[0], comp.slice(1).join(' ')))
+        var number = document.querySelector('#value').value
+        var select = document.getElementById('unit')
+        var index = select.selectedIndex
+        var selectedUnit = select.querySelectorAll('option')[index].innerHTML
+        var unit = ''
+        if (selectedUnit === 'wei') { unit = 'wei' }
```

Comment:
Well I don't really get why you need an extra variable here... could this be only one `unit` variable?

Reply:
The one is the value read from the UI element, the other the input passed to the library, I would recommend to keep this separate.

Reply:
(then it is more explicit/readable what is actually passed, and you are more flexible on naming/renaming things in the UI)